modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/AbstractEventEndpoint.java

```diff
@@ -2018,6 +2019,10 @@ public Response getNewMetadata() {
         collection.removeField(collection.getOutputFields().get("startTime"));
       if (collection.getOutputFields().containsKey("location"))
         collection.removeField(collection.getOutputFields().get("location"));
+      if (collection.getOutputFields().containsKey(DublinCore.PROPERTY_PUBLISHER.getLocalName())) {
+        MetadataField<String> publisher = (MetadataField<String>) collection.getOutputFields().get(DublinCore.PROPERTY_PUBLISHER.getLocalName());
+        publisher.setValue(getSecurityService().getUser().getName());
```

Comment:
This would overwrite an existing value, right? Maybe the field should be checked if it has a (default) value first?

Reply:
The event is new, so the field is still empty. The logged user is assigned directly.

Reply:
When the event is already created and you see its properties, then you can read the publisher because it already has a value. That is in the EventUtils.java class. Thanks for the review.

Reply:
Theoretically there could be a default value configured, though. This currently only works for metadata fields that are fed by a listprovider (e.g. _license_) but I haven't given up hope that one day a default can be configured for all metadata fields...then it would be overwritten here.

Reply:
Publisher is also a list provider, but not static as licenses. You need a list provider because you can search events by publisher (there is a publisher filter). Only users who have created events are in the list. But as event property of the wizard, a list has no sense in this case.

It would be possible to take a default value from a properties configuration file, for example, org.opencastproject.adminui.endpoint.OsgiEventEndpoint.cfg, and programatically fill the property first time with this value.

Reply:
+1